src/Types/ContextFactory.php

```diff
@@ -47,7 +47,7 @@ public function createFromReflector(\Reflector $reflector)
         $fileName = $reflector->getFileName();
         $namespace = $reflector->getNamespaceName();
 
-        if (file_exists($fileName)) {
+        if ($fileName && file_exists($fileName)) {
```

Comment:
could you make this more explicit, I generally don't trust the php casting model to booleans. 
`$fileName !== false`  

I think it would be nice to have a test for this as well.

Reply:
Well more correct would be `is_strict()`, since `file_exists(true)` fails as well.